packages/tideline/js/plot/confidentialModeEvent.js

```diff
@@ -85,9 +85,10 @@ module.exports = function (pool, options = {}) {
       selection.selectAll('.d3-confidential-group').on('mouseover', function (d) {
         const {duration} = utils.getDuration(d);
         if ( duration < maxSizeWithTooltip) {
-          const parentContainer = document.getElementsByClassName('patient-data')[0].getBoundingClientRect();
+          const parentContainer = document.getElementById('tidelineMain').getBoundingClientRect();
           const container = this.getBoundingClientRect();
           container.y = container.top - parentContainer.top;
+          container.x = container.left - parentContainer.left;
```

Comment:
does it worth moving this code into plot/utils/utils.js as a function like 
```
  getContainer: function(d) {
    const parentContainer = document.getElementById('tidelineMain').getBoundingClientRect();
    const container = d.getBoundingClientRect();
    container.y = container.top - parentContainer.top;
    container.x = container.left - parentContainer.left;
    return container;
  }
```
so that it can be called everywhere the same way as 
```
          confidentialModeEvent.addTooltip(d, utils.getContainer(this));
```

Reply:
Yes probably.
I didn't invest too much time on it, I will do it